tests/ops/test_qubit_ops.py

```diff
@@ -1530,42 +1530,6 @@ def test_U3_gate(self, tol):
 
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
-    def test_qubit_unitary(self, tol):
-        """Test that the unitary operator produces the correct output."""
-        U = np.array([[1, 1], [1, -1]]) / np.sqrt(2)
-        out = qml.QubitUnitary(U, wires=0).matrix
-
-        # verify output type
-        assert isinstance(out, np.ndarray)
-
-        # verify equivalent to input state
-        assert np.allclose(out, U, atol=tol, rtol=0)
-
-    def test_qubit_unitary_exceptions(self):
-        """Tests that the unitary operator raises the proper errors."""
-        U = np.array([[1, 1], [1, -1]]) / np.sqrt(2)
-
-        # test non-square matrix
-        with pytest.raises(ValueError, match="must be a square matrix"):
-            qml.QubitUnitary(U[1:], wires=0).matrix
-
-        # test non-unitary matrix
-        U3 = U.copy()
-        U3[0, 0] += 0.5
-        with pytest.raises(ValueError, match="must be unitary"):
-            qml.QubitUnitary(U3, wires=0).matrix
-
-    @pytest.mark.parametrize(
-        "U", [np.array([0]), np.array([1, 0, 0, 1]), np.array([[[1, 0], [0, 1]]])]
-    )
-    def test_qubit_unitary_not_matrix_exception(self, U):
```

Comment:
I removed this test because the case is already covered in the `unitary_exceptions` test, where `U[1:]` is passed as input.

Reply:
Nice :+1: